gateway/src/api/latest.rs

```diff
@@ -356,6 +366,15 @@ pub mod tests {
             .await
             .unwrap();
 
+        // delete returns 404 for project that doesn't exist
+        router
+            .call(delete_project("resurrections").with_header(&authorization))
```

Comment:
Can we also add a test to delete "reloaded" twice? (it would more closely cover your code change)

Reply:
Good point, I forgot to test that case. I added it now to the `end_to_end` test in `lib.rs`, since the test in `latest` doesn't progress the tasks.